app/src/main/java/edu/grinnell/appdev/grinnelldirectory/Activity/LoginActivity.java

```diff
@@ -70,7 +69,7 @@ void signIn(View view) {
             Intent intent = new Intent(getBaseContext(), MainActivity.class);
             startActivity(intent);
         } else {
-            AlertDialog.Builder builder = new AlertDialog.Builder(getApplicationContext());
+            AlertDialog.Builder builder = new AlertDialog.Builder(this);
```

Comment:
You should follow the *DRY* (Don't Repeat Yourself) principle. You should extract the lines here as a method with the relevant parameters and call the methods in all three cases. 

Also, you might as well `final`ize the `AlertDialog.Builder`